src/purify.js

```diff
@@ -1183,13 +1182,14 @@ function createDOMPurify(window = getGlobal()) {
   };
 
   /**
-   * _basicCustomElementCheck
+   * _isBasicCustomElement
    * checks if at least one dash is included in tagName, and it's not the first char
    * for more sophisticated checking see https://github.com/sindresorhus/validate-element-name
    *
    * @param {string} tagName name of the tag of the node to sanitize
+   * @returns {boolean} Returns true if the tag name meets the basic criteria for a custom element, otherwise false.
    */
-  const _basicCustomElementTest = function (tagName) {
+  const _isBasicCustomElement = function (tagName) {
```

Comment:
The function name in the jsdoc was different from the actual function name, so I tried to modify it, and finally changed it to _isBasicCustomElement with the is prefix because it returns a boolean.

jsdoc: _basicCustomElementCheck
actual: _basicCustomElementTest